tools/src/main/java/org/apache/kafka/trogdor/workload/Payload.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.trogdor.workload;
+
+import java.util.Random;
+
+/**
+ * Describes the payload for the producer record. Currently, we can configure message size
+ * (constant), expected compression rate. Where value is of message size, and key is null.
+ * We will likely make this class a base class in the future and derive different payload classes
+ * for various message size distributions, key assignments, etc.
+ */
+public class Payload {
+
+    public static final double DEFAULT_EXPECTED_COMPRESSION_RATE = 0.3;
+    public static final int DEFAULT_MESSAGE_SIZE = 512;
+
+    private final double expectedCompressionRate;
```

Comment:
Hmm.  We don't really know what the compression rate will be (that depends on the compression algorithm chosen).  Also, since we're using ASCII text, it will probably be somewhat compressible even there is no padding.

Maybe this should be called something like "fullRatio" to be a little more clear?

Reply:
Yes,  it's very approximate. It's not really empty or full ratio, because we initialize `byte[]` to random bytes and then change only a ratio of them. So that some bytes remain the same between messages and can be compressed. Maybe I could name it "similarityRatio".